bin/options.js

```diff
@@ -17,9 +17,13 @@ module.exports = getOptions;
  */
 
 function getOptions () {
-  var optsPath = process.argv.indexOf('--opts') === -1
-    ? 'test/mocha.opts'
-    : process.argv[process.argv.indexOf('--opts') + 1];
+  var defaultPath = 'test/mocha.opts';
+
+  var optsIndex = process.argv.indexOf('--opts');
+  var optsPath = optsIndex === -1
+    ? defaultPath
+    : process.argv[optsIndex + 1];
+  var shouldThrowError = optsIndex !== -1;
```

Comment:
I'd rename this to `wasOptsProvided`. It makes the code / reasoning more clearer imo.